api/v1alpha1/temporalworker_webhook_test.go

```diff
@@ -2,102 +2,132 @@
 //
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2024 Datadog, Inc.
 
-package v1alpha1
+package v1alpha1_test
```

Comment:
FYI: this is needed in order to avoid import cycles after moving test helpers to a separate package.